rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc

```diff
@@ -27,9 +27,21 @@ Machine pools are a higher level construct to compute machine sets.
 
 A machine pool creates compute machine sets that are all clones of the same configuration across availability zones. Machine pools perform all of the host node provisioning management actions on a worker node. If you need more machines or must scale them down, change the number of replicas in the machine pool to meet your compute needs. You can manually configure scaling or set autoscaling.
 
-By default, a cluster is created with one machine pool. You can add additional machine pools to an existing cluster, modify the default machine pool, and delete machine pools.
+By default, a cluster has one machine pool. During cluster installation, you can label this machine pool. After the cluster is installed, the `Default` machine pool cannot be deleted, and its node type or size cannot be changed.
 
-Multiple machine pools can exist on a single cluster, and they can each have different types or different size nodes.
+After a cluster's installation:
+
+* You can relabel a `Default` machine pool.
+* You can add additional machine pools to an existing cluster.
+* You can add a label to each added machine pool.
++
+[NOTE]
+====
+You cannot change the machine pool node type or size. The machine pool node type or size is specified during their creation only. After cluster installation, you cannot change the node type or size for the machine pool. If you need a different node type or size, you must re-create a machine pool and specify the required node type or size values.
+====
```

Comment:
@smunje1 Do you use a label to set the node type or size? If not, can you remove this note? It makes it seem like you can, as it is immediately after the _You can add a label_ bullet. Also, the paragraph before the list states the same thing already.

Reply:
@mburke5678 I have updated the Note. Could you please take a look and confirm? Thank you!